ads/_config.js

```diff
@@ -727,6 +727,11 @@ export const adConfig = {
 
   'postquare': {},
 
+  'pressboard': {
+    // prefetch: 'https://adserver.pressboard.ca/js/amp-ad.js',
```

Comment:
Why comment out? Do we want them?

Please also consider adopting the `renderStartAPI` to improve user experience. More information can be found [here](https://github.com/ampproject/amphtml/blob/master/ads/README.md#available-apis). Please let us know if there's any question. Thanks

Reply:
implemented renderStart, removed comments